tests/code/test_events.py

```diff
@@ -30,54 +29,26 @@
 from app.logic.users import addUserInterest, removeUserInterest, banUser
 from app.logic.utils import format24HourTime
 
-@pytest.mark.integration
-def test_event_model():
-
-    # single program
-    event = Event.get_by_id(12)
-    assert event.singleProgram == Program.get_by_id(3)
-
-    # no program
-    event = Event.get_by_id(13)
-    assert event.singleProgram == None
-    assert event.noProgram
-
-    # multi program
-    event = Event.get_by_id(14)
-    assert event.singleProgram == None
-    assert not event.noProgram
-
-    # program/event passed
-    event = Event.get_by_id(11)
-    assert event.isPast
+# @pytest.mark.integration
+# def test_event_model():
 
-@pytest.mark.integration
-def test_getAllEvents():
-
-    # No program is given, get all events
-    events = getEvents()
-
-    assert len(events) > 0
+#     # # single program
+#     # event = Event.get_by_id(12)
+#     # assert event.program == Program.get_by_id(3)
 
-    assert events[0].description == "Empty Bowls Spring 2021"
-    assert events[1].description == "Will donate Food to Community"
-    assert events[2].description == "Lecture on adoption"
-
-@pytest.mark.integration
-def test_getEventsWithProgram():
+#     # no program
+#     event = Event.get_by_id(13)
+#     assert event.program == None
+#     assert event.noProgram
 
-    # Single program
-    events = getEvents(program_id = 2)
+#     # multi program
+#     event = Event.get_by_id(14)
+#     assert event.program == None
+#     assert not event.noProgram
 
-    assert len(events) > 0
-    assert events[0].description == "Berea Buddies First Meetup"
-
-@pytest.mark.integration
-def test_getEventsInvalidProgram():
-
-    # Invalid program
-    with pytest.raises(DoesNotExist):
-        getEvents(program_id="asdf")
+#     # program/event passed
+#     event = Event.get_by_id(11)
+#     assert event.isPast
```

Comment:
Lots of deletions and comments here. Looks like there are some legit tests that are commented out.